### PR TITLE
Update tests so they will fail less

### PR DESCRIPTION
**Pull request by u9g**

### CODE REVIEW

**Comment by u9g** [comment]
I changed how the login message is awaited now running tests

**Comment by u9g** [comment]
10 tests and no fails, I feel confident that this made tests more stable

File: docs/README.md [hunk 1]
```diff
@@ -10,6 +10,8 @@ flying-squid
 
 Create Minecraft servers with a powerful, stable, and high level JavaScript API.
 
+Retry9
```
Comment (rom1504) [inline docs/README.md:13]: Can you remove this line and let's merge

**Comment by u9g** [comment]
done

**Review by rom1504 — APPROVED** [review]

**Pull request merged**